### PR TITLE
add koordinator scheduler charts

### PR DESCRIPTION
**Pull request by allwmh**
add koordinator scheduler charts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ helm repo add koordinator https://koordinator.github.io/charts
 ## Contributing
 
 We'd love to have you contribute! Please refer to our [contribution guidelines](CONTRIBUTING.md) for details.
+¡
```
Comment (eahydra) [inline README.md:14]: typo?

File: versions/v0.1.0/templates/koord-scheduler-config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            - name: cpu
+              weight: 1
+            - name: memory
+              weight: 1
+            - name: "koordinator.sh/batch-cpu"
+              weight: 10
+            - name: "koordinator.sh/batch-mem"
+              weight: 10
+          name: NodeResourcesBalancedAllocationArgs
+        schedulerName: koord-scheduler
```
Comment (eahydra) [inline versions/v0.1.0/templates/koord-scheduler-config.yaml:25]: Are these configurations sufficient for the scheduler to run? Is there some other built-in plugin configuration missing?

**Review by eahydra — CHANGES_REQUESTED** [review]

File: versions/v0.1.0/templates/koord-scheduler-config.yaml (hunk 2 above)
Comment (allwmh) [reply versions/v0.1.0/templates/koord-scheduler-config.yaml:25]: This config just change "NodeResourcesBalancedAllocation" plugin to support koordinator slo resources, other plugins will run the way it was.

**Review by stormgbs — APPROVED** [review]

**Pull request merged**